pkg/core/runtime/component/component.go

```diff
@@ -96,14 +96,14 @@ func (cm *manager) startLeaderComponents(stop <-chan struct{}, errCh chan error)
 	closeLeaderCh := func() {
 		mutex.Lock()
 		defer mutex.Unlock()
-		if channels.IsClosed(leaderStopCh) {
+		if !channels.IsClosed(leaderStopCh) {
```

Comment:
Am I right we Kuma CP didn't lose leadership because of this line?

Reply:
you are right.